.gitignore

```diff
@@ -72,4 +72,5 @@ lint/tmp/
 .DS_Store
 # App Specific cases 
 app/release/output.json
-.idea/codeStyles/
\ No newline at end of file
+.idea/codeStyles/
+/app/src/main/res/values/api-keys.xml
```

Comment:
Wszystko fajnie tylko jeśli to jest plik, który ty stworzyłeś, używasz zmiennych zdefiniowanych tam i nie ma go w repo, to ciężko bedzie nam to odpalić. Jeśli to nie jest klucz do jakiegoś super ważnego API, który jest płatny to go dodaj tutaj albo wyślij nam ten plik.